app/main/RTMedia.php

```diff
@@ -1496,6 +1496,49 @@ function rtmedia_get_site_option( $option_name, $default = false ) {
 	return $return_val;
 }
 
+function rtm_privacy_message_on_website() {
+	global $rtmedia;
+	$options = $rtmedia->options;
+
+	if( "1" === $options['general_upload_terms_show_pricacy_message'] && empty( $_COOKIE[ 'rtm_show_privacy_message' ] ) ) {
+		echo "<div class='privacy_message_wrapper'><p>" . wp_kses_post( $options[ 'general_upload_terms_privacy_message' ] ) . "</p><span class='dashicons dashicons-no' id='close_rtm_privacy_message'></span></div>";
+	}
+}
+add_action( 'wp_footer', 'rtm_privacy_message_on_website' );
+
+/**
+ * Function to add privacy policy information in WordPress policy section.
+ */
+function rtm_plugin_privacy_information() {
+    $policy = '';
+    if ( function_exists( 'wp_add_privacy_policy_content' ) ) {
+		$policy .= esc_html__( 'We collect your information during the checkout process on your purchase. The information collected from you may include, but is not limited to, your name, billing address, shipping address, email address, phone number, credit card/payment details and any other details that might be requested from you for the purpose of processing.', 'buddypress-media' );
+		$policy .= '<br/><br/><b>';
```

Comment:
can you please add this in sperate file or template, or check if can use ob_start()
this concatenation doesn't look good.